dist/avatar/avatar.css

```diff
@@ -1,17 +1,34 @@
 .avatar {
     align-items: center;
-    background-color: #84b4fb;
     border-radius: 50%;
-    color: #002a69;
+    display: flex;
     display: inline-flex;
     font-size: var(--font-size-large-2);
     font-weight: var(--font-weight-bold);
     height: 48px;
     justify-content: center;
     line-height: 48px;
     overflow: hidden;
+    position: relative;
     width: 48px;
 }
+.avatar:after {
```

Comment:
This is a pretty big shift on the base of `avatar`. I'm just wondering why `:after` is necessary and what limitations we have that makes it impossible to accomplish this in `.avatar`.

Reply:
This is applying an `image-treatment` to the avatar. Basically its masking the background of avatar. This is mainly in the case of an image that doesn't cover the whole avatar. 
If there is a background-color this won't be applied or affect the avatar.  
Current skin site:
<img width="647" alt="Screenshot 2025-02-11 at 9 29 23 AM" src="https://github.com/user-attachments/assets/807ebfe2-511b-4f59-ba32-64a5f79a7a1f" />
Localhost:
<img width="619" alt="Screenshot 2025-02-11 at 9 29 31 AM" src="https://github.com/user-attachments/assets/2eb4c370-3b63-42f5-ace8-bacb321303c0" />

Reply:
Sounds good. Thanks!